packages/core-svelte/src/components/TextDropdownComponent.svelte

```diff
@@ -213,7 +213,7 @@
                     }
                 }
             }
-        } else { // this component was entered using keystrokes, no dropdown is shown
```

Comment:
Why change a comment to something with less info? Is that amn improvement?

Reply:
I do not understand the text up until the comma, because this is an onKey event handler and everything is entered using keystrokes.

Reply:
Ok. Maybe change the comment into something that is more understandable: "this component was selected using keystrokes, not by clicking, therefore dropDownShown  = false".